test/e2e/storage/testsuites/base.go

```diff
@@ -325,14 +325,19 @@ func createVolumeSourceWithPVCPVFromDynamicProvisionSC(
 	pvc, err = cs.CoreV1().PersistentVolumeClaims(ns).Create(pvc)
 	Expect(err).NotTo(HaveOccurred())
 
-	err = framework.WaitForPersistentVolumeClaimPhase(v1.ClaimBound, cs, pvc.Namespace, pvc.Name, framework.Poll, framework.ClaimProvisionTimeout)
-	Expect(err).NotTo(HaveOccurred())
+	if !isDelayedBinding(sc) {
```

Comment:
do callers of `createVolumeSourceWithPVCPVFromDynamicProvisionSC()` handle nil PVC and PVs gracefully?

Reply:
at least in this file yes.  I'm currently running all of the tests to see if I can find other issues